gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/util/GitHub.kt

```diff
@@ -73,7 +72,8 @@ internal object GitHub {
       "File $gitHubEventPath doesn't exist"
     }
 
-    return json.decodeFromStream(file.inputStream())
```

Comment:
This is an experimental API which we shouldn't use in a public plugin. It was causing very bizarre crashes until I noticed what was happening.